src/embedding.jl

```diff
@@ -43,54 +40,34 @@ function embedding_map(λCnt::Int, encoding::Any=ebd_gray, ibs::Bool=false)
 	return map
 end
 
-"""
-	This function parse the encoding key string and return the built-in encoding function.
-"""
 function resolve_encoding_key(encoding::Any)
 	isa(encoding, Function) && return encoding
 	encoding == "default" && return ebd_gray
-	encoding == "gray" && return ebd_gray
-	encoding == "binary" && return ebd_binary
-	error("Must specify a encoding method when using embedding formulation")
+	error("Must specify a encoding method when using convhull_ebd formulation")
 end
 
 
-"""
-	This function is the same σ() function described in Vielma and Nemhauser 2011.
-"""
 function ebd_σ(b::String)
```

Comment:
Why were the comments removed?

Reply:
Sorry. This was a mistake. I have to hand code the changes between PRs to keep track of changes.